modules/tests/stress.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+. ./defaults.sh
+. ../../include/common.sh
+. .envrc
+
+STRESS_FOLDER=$(basename "$STRESS_TEST_REPO")
+
+[ ! -d "$STRESS_FOLDER" ] && git clone --recurse-submodules "$STRESS_TEST_REPO" "$STRESS_FOLDER"
+
+pushd "$STRESS_FOLDER" || exit
+
+if [ -n "$EKCP_PROXY" ]; then
+    export https_proxy=socks5://127.0.0.1:${KUBEPROXY_PORT}
+fi
+
+bash ./start_benchmark.sh
```

Comment:
Nitpicking, the bash call shouldn't be needed:

```suggestion
./start_benchmark.sh
```